MonoGame.Framework/TextInputEventArgs.cs

```diff
@@ -8,25 +8,30 @@
 namespace Microsoft.Xna.Framework
 {
     /// <summary>
-    /// This class is used for the game window's TextInput event as EventArgs.
+    /// An event containing keyboard text input data.  See
+    /// <see cref="GameWindow.TextEvent"/> for details.
     /// </summary>
     public class TextInputEventArgs : EventArgs
     {
-        char character;
-        public TextInputEventArgs(char character, Keys key = Keys.None)
```

Comment:
I am aware of that...  I didn't want to let it pass that way.  I wanted to fix the other platforms that use the event to have matching behavior to your changes in DesktopGL.